worker/peergrouper/worker_test.go

```diff
@@ -762,29 +770,29 @@ func (s *workerSuite) TestWorkerPublishesInstanceIds(c *gc.C) {
 }
 
 // mustNext waits for w's value to be set and returns it.
-func mustNext(c *gc.C, w *voyeur.Watcher) (val interface{}) {
+func mustNext(c *gc.C, w *voyeur.Watcher, context string) (val interface{}) {
```

Comment:
mustNext was logging a pointer to the Watcher object, but that doesn't give you any understanding of what is going on. Change it to use a context variable instead.